com.unity.render-pipelines.core/Runtime/Common/DynamicResolutionHandler.cs

```diff
@@ -246,21 +247,28 @@ public Vector2Int GetScaledSize(Vector2Int size)
                 return size;
             }
 
-            float scaleFractionX = m_CurrentFraction;
-            float scaleFractionY = m_CurrentFraction;
-            if (!m_ForceSoftwareFallback && type == DynamicResolutionType.Hardware)
-            {
-                scaleFractionX = ScalableBufferManager.widthScaleFactor;
-                scaleFractionY = ScalableBufferManager.heightScaleFactor;
-            }
+            Vector2Int scaledSize = ApplyScalesOnSize(size);
+            m_LastScaledSize = scaledSize;
+            return scaledSize;
+        }
 
-            Vector2Int scaledSize = new Vector2Int(Mathf.CeilToInt(size.x * scaleFractionX), Mathf.CeilToInt(size.y * scaleFractionY));
+        /// <summary>
+        /// Applies to the passed size the scale imposed by the dynamic resolution system.
+        /// Note: this function is pure (has no side effects).
+        /// </summary>
+        /// <param name="resolvedScales">Scale amount to use. Optionally, could utilize GetScaledSize function to get the scales desired, but this function is safer
```

Comment:
Not sure about the safer, as the idea is that if a custom SRP uses the system, then the GetScaleSize side effect is needed to keep the system functioning as long as it is called at the right time. 

Also no parameter here is resolvedScales in the function signature.  

I'd expand the API doc here to say that this doesn't cache the last scale size or the pre-scale size (i.e. the cachedOriginalSize )

Reply:
fixed documentation. Done!